modules/manage/partials/tiered-storage.adoc

```diff
@@ -1133,6 +1133,65 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true
 
 See also: xref:{topic-recovery-link}[Topic Recovery], xref:manage:kubernetes/k-remote-read-replicas.adoc[Remote Read Replicas]
 
+== Safe pause and resume
+
+Starting in version 25.1, when running Tiered Storage, you can safely pause and resume uploads to cloud storage without risking data consistency or loss. To pause or resume segment uploads to cloud storage, use the `cloud_storage_enable_segment_uploads` configuration property (default is `true`), which allows segment uploads to proceed normally after the pause completes and uploads resume.
```

Comment:
We should say when and how this will be used:

"Starting in version 25.1, when running Tiered Storage, to troubleshoot and resolve temporary issues related to a cluster's interaction with cloud storage, you can safely ..."

And, just in case this does end up in docs, let's also have an yellow box warning "We highly recommend using pause and resume only under the guidance of Redpanda support / customer success"